app/assets/src/components/layout/divider.scss

```diff
@@ -2,5 +2,5 @@
 
 .divider {
   height: 0px;
-  border-bottom: 8px solid $off-white;
+  border-bottom: 2px solid $off-white;
```

Comment:
I think this breaks the styling on the Sample Report tabs. Since the blue underline is still 8px

Reply:
I confirmed that this is not affecting the report page.